src/components/apps/dungeon/css/footer.css

```diff
@@ -0,0 +1,15 @@
+  .live-feed-txt{
```

Comment:
are these being used somewhere?

Reply:
that would be good, obviously happy to have them added back later if/when they are used

Reply:
@daoplays Removed the files in the latest commit